doc/kyuafile.5.in

```diff
@@ -173,6 +173,38 @@ Refer to the
 section below for clarification.
 .It Va description
 Textual description of the test.
+.It Va execenv
+The name of the execution environment to be used for running the test.
+If empty or not defined, the
+.Sq host
+execution environment is meant.
+The possible values are:
+.Bl -tag -width xUnnnnnnn
+.It host
+The default environment which runs the test as a usual child process.
+.It jail
+The
+.Fx
+jail environment.
```

Comment:
`.Xr jail 8` feels like the best one due to kyua users are expected to be usual users of `jail(8)`, instead of the system calls and the jail library. And the `execenv.jail` parameters are actually parameters for `jail(8)`. What do you think?

Reply:
I've put `.Xr jail 8` for now.